packages/storybook/stories/index.tsx

```diff
@@ -132,20 +131,24 @@ export const {
 });
 
 export const FolderTabs = Tabs;
-export const UnderlineTabs = createTabs(defaultConfigs.underlineTabs);
+export const UnderlineTabs = withBentoConfig({ tabs: defaultConfigs.underlineTabs }, Tabs);
 export const RightActions = Actions;
-export const LeftActions = createActions(
+export const LeftActions = withBentoConfig(
   {
-    ...defaultConfigs.actions,
-    buttonsAlignment: "left",
-    primaryPosition: "left",
+    actions: {
+      ...defaultConfigs.actions,
+      buttonsAlignment: "left",
+      primaryPosition: "left",
+    },
   },
-  { Button, Banner, InlineLoader }
+  Actions
 );
-export const SpaceBetweenActions = createActions(
+export const SpaceBetweenActions = withBentoConfig(
   {
-    ...defaultConfigs.actions,
-    buttonsAlignment: "spaceBetween",
+    actions: {
+      ...defaultConfigs.actions,
+      buttonsAlignment: "spaceBetween",
+    },
   },
-  { Button, Banner, InlineLoader }
+  Actions
```

Comment:
`withBentoConfig` is a utility for wrapping a component in a custom `BentoConfigProvider`, so that we can create different variants of a component.